examples/managed-identity/kubernetes.json

```diff
@@ -0,0 +1,38 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "useManagedIdentity": true

```

Comment:
@shrutir25  I think it's going to stay here a while longer. There's no need to rush it and I have a better solution than the `hack/` stuff with the Azure function coming in ~3 weeks via ARM templates.